core/mm/vm.c

```diff
@@ -554,6 +555,10 @@ static bool cmp_region_for_remap(const struct vm_region *r0,
 	       r0->mobj == r->mobj && rn->offset == r->offset + r->size;
 }
 
+/*
+ * NOTE: `new_va` always belongs to userland. Hence, it is safe to always use
```

Comment:
If I'm right, 'hence' is not expected to be used to start a sentence. Prefer use 'therefore', unless someone argues i'm wrong :)